engine/src/main/java/org/camunda/bpm/engine/impl/util/ReflectUtil.java

```diff
@@ -331,44 +328,21 @@ private static Method findMethod(Class< ? extends Object> clazz, String methodNa
     return null;
   }
 
-  public static Object instantiate(String className, Object[] args) {
```

Comment:
The method is used in several places (e.g. [here](https://github.com/camunda/camunda-bpm-platform/blob/master/engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/hal/cache/HalRelationCacheBootstrap.java), [here](https://github.com/camunda/camunda-bpm-platform/blob/master/engine/src/main/java/org/camunda/bpm/engine/impl/form/validator/DelegateFormFieldValidator.java), and [here](https://github.com/camunda/camunda-bpm-platform/blob/master/engine/src/main/java/org/camunda/bpm/container/impl/deployment/StartProcessEngineStep.java)), if I am not mistaken. So I guess we cannot remove it.

Reply:
Fair point. I only checked in the IDE. Should have consulted the full codebase on GitHub :D